src/callpathExample/callpathExample.st

```diff
@@ -0,0 +1,29 @@
+PROGRAM UpCounter
+    VAR_EXTERNAL
+        counterInt:       Int;
```

Comment:
CounterInt : INT --> style guide conform

Reply:
style guide applied - not only for this one